### ISSUE TITLE
Fix snabb-lwaftr run -vv

### ISSUE THREAD

**Issue by kbara** [Igalia/snabbswitch#213]
```
% snabb-lwaftr run 
Usage: run --help
       run --conf <conf-file> --v4-pci <pci-addr> --v6-pci <pci-addr> [-D <seconds>] [-v] [-u]

       --conf   <conf-file>     Sets configuration policy table
       --v4-pci <pci-addr>      PCI device number for the INET-side NIC
       --v6-pci <pci-addr>      PCI device number for the B4-side NIC
       -D       <seconds>       Duration in seconds
       -v                       Verbose (repeat for more verbosity)
```
Either we should fix -u (ultra-verbose) or remove it from the short help. I'd prefer the former.

**Comment by kbara** [comment]
It's apparently been replaced by -v -v, however:

```
snabb-lwaftr run --conf ./icmp_on_fail.conf --v4-pci 02:00.0 --v6-pci 02:00.1 -D  3 -v -v
loading compiled binding table from ./binding-table.o
compiled binding table ./binding-table.o is up to date.
Time (s),IPv4 RX MPPS,IPv4 RX Gbps,IPv4 TX MPPS,IPv4 TX Gbps,IPv6 RX MPPS,IPv6 RX Gbps,IPv6 TX MPPS,IPv6 TX Gbps
program/snabb_lwaftr/run/run.lua:95: variable 'app' is not declared
stack traceback:
	core/main.lua:124: in function <core/main.lua:122>
	[C]: in function 'error'
	lib/lua/strict.lua:37: in function '__index'
	program/snabb_lwaftr/run/run.lua:95: in function 'fn'
	core/timer.lua:33: in function 'call_timers'
	core/timer.lua:42: in function 'run_to_time'
	core/timer.lua:23: in function 'run'
	core/app.lua:238: in function 'main'
	program/snabb_lwaftr/run/run.lua:109: in function 'run'
	program/snabb_lwaftr/snabb_lwaftr.lua:17: in function 'run'
	core/main.lua:58: in function <core/main.lua:32>
	[C]: in function 'xpcall'
	core/main.lua:153: in main chunk
	[C]: at 0x00450390
	[C]: in function 'pcall'
	core/startup.lua:1: in main chunk
	[C]: in function 'require'
	[string "require "core.startup""]:1: in main chunk
```

**Comment by wingo** [comment]
The intention was to remove the -u option and replace with -vv or -v -v.  The short help should be updated accordingly and obviously the bug should be fixed :)

**Comment by wingo** [comment]
Fixed in the 2.0 release.
